### PR TITLE
Async testing fixes

### PR DESCRIPTION
**Pull request by knodwell**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/ws/rabbit/LaunchReporterConsumer.java [hunk 1]
```diff
@@ -52,36 +57,92 @@
...
+				});
+
+				return;
+			}
+			LOGGER.trace("Retrying start request for Launch {}, attempt {}", rq.getUuid(), count);
+		}
+		try {
+			ReportPortalUser userDetails = (ReportPortalUser) userDetailsService.loadUserByUsername(username);
+			startLaunchHandler.startLaunch(userDetails, ProjectExtractor.extractProjectDetails(userDetails, projectName), rq);
+		} catch (Exception e) {
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/ws/rabbit/LaunchReporterConsumer.java]: let's configure exception logging on some spring (if possible) or aop layer

Comment (knodwell) [reply src/main/java/com/epam/ta/reportportal/ws/rabbit/LaunchReporterConsumer.java]: it's high time to do, added )

**Comment by avarabyeu** [comment]
guys, is it supposed to be merged?

**Comment by knodwell** [comment]
> guys, is it supposed to be merged?
no, closing it.
I will clean up old async PR and branches

**Pull request closed without merging**